apigee_m10n.install

```diff
@@ -35,15 +35,20 @@ function apigee_m10n_requirements($phase) {
 
   // If monetization is disabled after module has been installed, show error on status page.
   if ($phase === 'runtime') {
-    /** @var \Drupal\apigee_m10n\MonetizationInterface $monetization */
-    $monetization = Drupal::service('apigee_m10n.monetization');
-
-    if (!$monetization->isMonetizationEnabled()) {
-      $requirements['apigee_monetization_connection_error'] = [
-        'title' => t('Apigee Monetization'),
-        'description' => t(Monetization::MONETIZATION_DISABLED_ERROR_MESSAGE),
-        'severity' => REQUIREMENT_ERROR,
-      ];
+    try {
+      /** @var \Drupal\apigee_m10n\MonetizationInterface $monetization */
+      $monetization = Drupal::service('apigee_m10n.monetization');
+
+      if (!$monetization->isMonetizationEnabled()) {
+        $requirements['apigee_monetization_connection_error'] = [
+          'title' => t('Apigee Monetization'),
+          'description' => t(Monetization::MONETIZATION_DISABLED_ERROR_MESSAGE),
+          'severity' => REQUIREMENT_ERROR,
+        ];
+      }
+    }
+    catch (Exception $exception) {
+
```

Comment:
should probably do something with the exception, maybe log to watchdog?

Reply:
I don't think we want to just hide all exceptions here. We should probably be catching `\Apigee\Edge\Exception\ApiException` here.

Reply:
Done and done.